service_mesh/v2x/preparing-ossm-installation.adoc

```diff
@@ -15,6 +15,8 @@ Before you can install {ProductName}, review the installation activities, ensure
 ** xref:../../installing/installing_aws/installing-aws-user-infra.adoc#installing-aws-user-infra[Install {product-title} {product-version} on user-provisioned AWS]
 ** xref:../../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metal[Install {product-title} {product-version} on bare metal]
 ** xref:../../installing/installing_vsphere/installing-vsphere.adoc#installing-vsphere[Install {product-title} {product-version} on vSphere]
+** xref:../../installing/installing_ibm_z/installing_ibm_z.adoc#installing-ibm-z[Install {product-title} 4.6 on IBM Z and LinuxONE]
```

Comment:
The standard in OpenShift is underscores (_) in folder names, and dashes (-) in assembly and module file names.  It looks like you've got that correct in the folders and files, but put all underscores in the XREF, which is why the build is failing.